pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/TopicTransactionBuffer.java

```diff
@@ -674,6 +700,10 @@ private void closeReader(SystemTopicClient.Reader<TransactionBufferSnapshot> rea
         }
     }
 
+    public interface MaxReadPositionCallBack {
+        void moveForward(PositionImpl oldPosition, PositionImpl newPosition);
```

Comment:
`moveForward` is in imperative format and it sounds like a command. It would be better to rename it so that it's clear that this is an event. `maxReadPositionMovedForward` would be one possibility. 
Please also add Javadoc that clarifies the purpose and contract of this interface and the event information that the method call provides.

Reply:
I have change the method name and complete the documents, PTAL, thanks.